pyg_lib/csrc/sampler/cpu/neighbor_kernel.cpp

```diff
@@ -637,21 +706,25 @@ std::tuple<at::Tensor,
            at::Tensor,
            c10::optional<at::Tensor>,
            std::vector<int64_t>,
+           std::vector<int64_t>,
            std::vector<int64_t>>
 neighbor_sample_kernel(const at::Tensor& rowptr,
```

Comment:
Are you okay with adding `distributed_neighbor_sample` function instead at a high-level? This way, we don't necessarily need to make changes to the interface for `distributed=False`. We can still use `DISPATCH_SAMPLE` internally in these functions.